Cargo.toml

```diff
@@ -142,7 +142,7 @@ icu_calendar = { version = "~2.0.0", path = "components/calendar", default-featu
 icu_casemap = { version = "~2.0.0", path = "components/casemap", default-features = false }
 icu_collator = { version = "~2.0.0", path = "components/collator", default-features = false }
 icu_collections = { version = "~2.0.0", path = "components/collections", default-features = false }
-icu_codepointtrie_builder = { version = "~0.4.0", path = "components/collections/codepointtrie_builder", default-features = false }
+icu_codepointtrie_builder = { version = "~0.5.0", path = "components/collections/codepointtrie_builder", default-features = false }
```

Comment:
we should changelog this as well, i missed this one because it's a subfolder

Reply:
I think it's fine, there are no changes except for the `icu_collections` version

Reply:
Yeah but we mention non-mainline versions in the changelog, usually.